CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Change Log for coreHTTP Client Library
+
+## v1.0.0 (MONTH YEAR)
+
+This is the first release of the coreHTTP client library in this repository.
+
+The HTTP client library is a client-side implementation that implements a subset
```

Comment:
An "implementation that implements"?  Maybe reword this?

Reply:
lol sure.